src/client/Components/Util/LogTable.jsx

```diff
@@ -1,12 +1,25 @@
+// @flow
+
 import React from 'react';
 
-export default class LogTable extends React.Component {
-  constructor(props) {
+type Props = {
+  github_repo: ?string
+};
+
+type State = {
+  github_repo: string,
+  isLoaded: boolean,
+  componentLogs: Array<ComponentLogItem>
+};
+
+export default class LogTable extends React.Component<Props, State> {
+  constructor(props: Props) {
     super(props);
 
     this.state = {
-      github_repo: this.props.github_repo,
-      isLoaded: false
+      github_repo: this.props.github_repo != null ? this.props.github_repo : '',
```

Comment:
Why is the empty string a valid piece of data here?

Reply:
Null data. What do you think would be a better alternative?

Reply:
If it’s genuinely nullable, it should be marked as such and dealt with explicitly. When can this be null anyway?

Reply:
I'll make it thrown an error instead